compass/machines/pm-gpu.cfg

```diff
@@ -31,6 +31,10 @@ spack = /global/cfs/cdirs/e3sm/software/compass/pm-gpu/spack
 # pnetcdf as E3SM (spack modules are used otherwise)
 use_e3sm_hdf5_netcdf = True
 
+# spack variants for Albany and Trilinos
+albany_variants = +mpas~py+unit_tests+cuda+uvm+sfad sfadsize=12
+trilinos_variants = +cuda+uvm+ampere80+zen3
```

Comment:
@mcarlson801, can you double check that these are what I should be using?

Reply:
Yep, this looks right to me. We could probably remove the `uvm` flag now though, what do you think @jewatkins?

Reply:
Looks like there might be some lingering issues with no-uvm. The configuration you have here is good for now.

Reply:
Thanks @mcarlson801!  I appreciate it!